### DISCUSSION TITLE
Translated Content

### DISCUSSION THREAD

**Discussion by just-another-alien** [UkrainianBanderasCat/Digital-Logic-Sim2#63] [Ideas]
Please add translated content in different languages, it will be very nice.
I can help translating English/German if you want. 
I hope it is not to much.

**Comment by AlphaCodingPilot** [comment]
There is not a lot that has to be translated. Only the buttons in the top bar and the built-in chips.

**Reply by just-another-alien** [reply]
And the menu

**Reply by UkrainianBanderasCat** [reply]
... I added localization
